### ISSUE TITLE
Call external editor to edit `executable-fold` and refresh

### ISSUE THREAD

**Issue by hammerfunctor** [XmacsLabs/mogan#1840]
### Is your feature request related to a problem? Please describe.

[0001-1.-Add-alternate-force-eval-and-wrap-definition-of-a.patch.txt](https://github.com/XmacsLabs/mogan/files/15070747/0001-1.-Add-alternate-force-eval-and-wrap-definition-of-a.patch.txt)

### Describe the solution you'd like

Call external editor to edit `executable-fold` and refresh. Useful for things like heavy tikz plotting.

### Describe alternatives you've considered

Forget about existing excellent editors and improve the code editing experience in TeXmacs. Aesthetic but not worth developer's time.

**Comment by da-liii** [comment]
> Aesthetic but not worth developer's time.

It's developers' joy!

**Comment by hammerfunctor** [comment]
> It's developers' joy!

Of course, I mean, developers have things much more joyful than this to do.
